docker-storage-setup.sh

```diff
@@ -990,6 +989,10 @@ setup_docker_root_lv_fs() {
 }
 
 check_storage_options(){
+  if [ "$STORAGE_DRIVER" == "devicemapper" ] && [ -z "$CONTAINER_THINPOOL" ];then
+     Fatal "CONTAINER_THINPOOL must be defined for the devicemapper storage driver."
+  fi
+
```

Comment:
One last thing. Can we move this check in create_lvm_thin_pool(). And reason being that we have shipped code where we can create a thin pool without passing CONTAINER_THINPOOL. I don't want these people to be broken over upgrade. 

So this check will kick in only if we are creating a new thin pool. If thin pool is already there, then this check should not kick in.